Makefile

```diff
@@ -4,6 +4,17 @@ IMG ?= quay.io/${USER}/odh-model-controller:latest
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.26
 
+ENGINE ?= docker
+#####################################
+# Podman only parameter
+ARCH ?= x86_64
```

Comment:
that could, but considering that we support only x86 atm, it might be ok.

but it is probably better to remove it and do the architecture stuff on a script.

Reply:
let's go without it for now.